ksqldb-engine/src/test/java/io/confluent/ksql/query/QueryExecutorTest.java

```diff
@@ -178,7 +179,7 @@
   @Mock
   private SessionConfig config;
   @Mock
-  private QueryMetadata.Listener queryListener;
+  private QueryMetadataImpl.ListenerImpl queryListener;
```

Comment:
Why we need to declare as `QueryMetadataImpl.ListenerImpl` here now?

Reply:
I just removed `ListenerImpl` so no